### PR TITLE
Promote comment to answer 

### LINKED ISSUE #3081: upgrade comments to answers
Hi! 
we previously had a feature to upgrade comments to answers on the question/answer system. Seems like we don't have that right now. Can we get it back :)? 

Best,
Stevie!

### PR DESCRIPTION
**Pull request by gauravano**
Fixes #3081 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by gauravano** [comment]
Hi @jywarren , promoting comment to answer done for both types of comments. Here's final view -  https://drive.google.com/open?id=1U2gEw0KIWSqM95jNFs4jpd4vI6aM2g24 . 

the last codeclimate issue is pretty confusing, could you help me with it? Thanks!

**Comment by jywarren** [comment]
I think it's saying showing a raw db column value is a little unsafe. If you want to sanitize it that could probably help, or we could ignore if we think it's not a big security risk.

File: app/views/questions/_answer.html.erb [hunk 1]
```diff
@@ -27,7 +27,7 @@
       </div>
     </div>
     <div id="c<%= answer.id %>show" style="padding:0px 20px">
-      <p><%= raw insert_extras(answer.body_markdown) %></p>
+      <p><%= sanitize insert_extras(answer.body_markdown) %></p>
```
Comment (gauravano) [inline app/views/questions/_answer.html.erb]: Sanitized! @jywarren

**Comment by jywarren** [comment]
Awesome!

**Pull request merged**